setup.py

```diff
@@ -28,13 +22,19 @@
     author_email = "help@twilio.com",
     url = "https://github.com/twilio/twilio-python/",
     keywords = ["twilio","twiml"],
-    install_requires = REQUIRES,
-    # bdist conditional requirements support
+    install_requires = [
+        "six",
+        "pytz",
+        "PyJWT >= 1.4.2, <1.5.1",
+    ],
     extras_require={
-        ':python_version=="3.3"': ['pysocks'],
-        ':python_version=="3.4"': ['pysocks'],
-        ':python_version=="3.5"': ['pysocks'],
-        ':python_version=="3.6"': ['pysocks'],
+        ':python_version=="2.7"': [
+            "requests[security] >= 2.0.0",
+        ],
+        ':python_version>="3.3"': [
+            "requests >= 2.0.0",
```

Comment:
Those come with `requests[security]`. That's why I split it up, because Python 2.7 needs the extra installation of `requests`, but Python >= 3.3 doesn't.